web/src/ProviderEditPage.js

```diff
@@ -82,6 +99,20 @@ class ProviderEditPage extends React.Component {
     });
   }
 
+  getCountryCodes() {
+    OrganizationBackend.getOrganizations("admin")
```

Comment:
Did you add the frontend part?

![image](https://user-images.githubusercontent.com/3787410/222458944-36a7206d-8ef7-4be9-8549-27b4162739a4.png)